src/main/resources/META-INF/syndesis/syndesis-extension-definition.json

```diff
@@ -1,13 +1,14 @@
 {
   "tags" : [ "experimental", "feature" ],
   "actions" : [ {
-    "actionType": "extension",
+    "actionType": "step",
     "id": "damage-report",
     "name": "Damage Reporter",
     "description": "Generates contact list for damaged items",
     "descriptor": {
       "kind": "ENDPOINT",
       "entrypoint": "direct:damage-report",
+      "resource": "classpath:META-INF/syndesis/extensions/extension.xml",
       "inputDataShape": {
         "kind": "java",
         "type": "java.lang.String",
```

Comment:
I think `specification` can be omitted and leave the plugin to generate it, ca you give it a try ?

Reply:
@lburgazzoli yes extension is generating dataShapes properly. I've pushed an update.